kolibri/plugins/facility/assets/src/views/ImportCsvPage/InfoModal.vue

```diff
@@ -42,8 +42,8 @@
             <div>{{ $tr('possibleValues') }}</div>
             <ul>
               <li><code>ADMIN</code></li>
-              <li><code>COACH</code></li>
-              <li><code>ASSIGNABLE_COACH</code></li>
+              <li><code>FACILITY_COACH</code></li>
+              <li><code>CLASS_COACH</code></li>
```

Comment:
note I already am making this change in another PR so you can remove this to avoid a conflict:

https://github.com/learningequality/kolibri/pull/6822/files#diff-f27530fa36d3478dddd8bfc53dd093b7R57-R58

Reply:
I have done a rebase to sync your changes